src/Roslyn.Suppressions/Roslyn.Suppressions.UnitTests/InProcess/RoslynSettingsFileSynchronizerTests.cs

```diff
@@ -154,6 +156,47 @@ public async Task UpdateFileStorage_FileStorageIsUpdatedOnBackgroundThread()
             roslynSettingsFileStorage.Verify(x => x.Update(It.IsAny<RoslynSettings>()), Times.Once);
         }
 
+        [TestMethod]
+        public async Task UpdateFileStorage_IssuesAreConvertedAndFiltered()
+        {
+            var configuration = CreateConnectedConfiguration("some project key");
+            var activeSolutionBoundTracker = CreateActiveSolutionBoundTracker(configuration);
+
+            var sonarIssues = new []
```

Comment:
Side comment: I think it would've been a bit cleaner and easier to read if we would just mock this, instead of repeating the details of the actual logic. But we can take a look at it next time.